pkg/datapath/iptables/iptables.go

```diff
@@ -1140,3 +1151,25 @@ func (m *IptablesManager) addCiliumNoTrackXfrmRules() error {
 	}
 	return nil
 }
+
+func (m *IptablesManager) addCiliumENIRules() error {
+	if err := runProg("iptables", append(
+		m.waitArgs,
+		"-t", "mangle",
+		"-A", ciliumPreMangleChain,
+		"-i", option.Config.EgressMasqueradeInterfaces,
+		"-m", "comment", "--comment", "cilium: primary ENI",
+		"-m", "addrtype", "--dst-type", "LOCAL", "--limit-iface-in",
+		"-j", "CONNMARK", "--set-xmark", "0x80/0x80"),
```

Comment:
Do we have a Golang representation of the common marks we use or do we just set them via strings like this everywhere? If the former, let's make this the same as the other code. If the latter, let's file an issue to follow up & collect these so we can more easily reason about them.

Reply:
It seems we have a representation for the other _marks_ (e.g. `markAsFromHost`), I have updated my code accordingly.

I suppose long-term we might also want to wrap the recurrent arguments a bit to avoid appending so many strings? Would you like me to open an issue for that?

Reply:
By the way most of those representations take the form
```go
markAsFromHost := fmt.Sprintf("%#08x/%#08x", linux_defaults.MagicMarkHost, linux_defaults.MagicMarkHostMask)
```
We could maybe at least add a wrapper to have 
```go
markAsFromHost := makeXmark(linux_defaults.MagicMarkHost, linux_defaults.MagicMarkHostMask)
```
instead.

Reply:
> I suppose long-term we might also want to wrap the recurrent arguments a bit to avoid appending so many strings? Would you like me to open an issue for that?

Long-term we want to drop the rules entirely ;) But mid-term, yeah seems like a reasonable aim to avoid extra appends.  Not a huge deal for relatively static generation of rules so I see it maybe more from the angle that it could improve readability.

Reply:
Will do the mid-term change as a follow-up.